src/btree/bt_debug.c

```diff
@@ -1638,6 +1652,12 @@ __debug_ref(WT_DBG *ds, WT_REF *ref)
     if (__wt_ref_addr_copy(session, ref, &addr) && !WT_TIME_AGGREGATE_IS_EMPTY(&addr.ta))
         WT_RET(ds->f(ds, ", %s, %s", __wt_time_aggregate_to_string(&addr.ta, time_string),
           __wt_addr_string(session, addr.addr, addr.size, ds->t1)));
+    if (ref->state == WT_REF_DELETED && ref->page_del != NULL) {
```

Comment:
@kommiharibabu, do we still need to check for the `state` here? With WT-9987, we prove that the information contained by `page_del` could be interesting when the page is in memory.

Reply:
Yes, we don't need to check for `WT_REF_DELETED`. Whenever the `page_del` is not NULL, dump the information.

Reply:
I removed the check